examples/dockerfiles/README.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for using the Dockerfiles
+
+To build these dockerfiles go to the Gluon-TS root directory and run:
```

Comment:
I think what also works is doing

    docker build ../.. -f <dockerfile>

in the current folder. Maybe we should list both ways.

Reply:
Added this